### PR TITLE
Add Cypress test to edit function from action table

### PR DESCRIPTION
**Pull request by petterwildhagen**
Add Cypress test for edit action from actions-table accessed from
- follow up
- landing page

### CODE REVIEW

File: frontend/cypress/integration/actions_spec.ts [hunk 1]
```diff
@@ -124,37 +127,78 @@ describe('Actions management', () => {
...
+                newNotes = actionTestdata.createNewNotes(updatedAction, notesCreator)
 
                 actionsGrid.actionLink(action.questionOrder, action.title).click()
-
-                editActionDialog.assertActionValues(action, actionNotes)
-                editActionDialog.editAction(updatedAction, newNotes, actionNotes.length)
-                editActionDialog.assertNotesValues(actionNotes.concat(newNotes))
-                editActionDialog.close()
-
+                editAction(action, actionNotes, updatedAction, newNotes)
```
Comment (maiaht) [inline frontend/cypress/integration/actions_spec.ts:153]: Maybe `editActionDialog.body().should('be.visible')` should be included in this test as well?

Comment (petterwildhagen) [reply frontend/cypress/integration/actions_spec.ts:154]: Fixed

File: frontend/cypress/integration/actions_spec.ts [hunk 2]
```diff
@@ -124,37 +127,79 @@ describe('Actions management', () => {
...
+                const assignedToParticipant = new Participant({
+                    user: user,
+                    role: faker.random.arrayElement(assignableRoles),
+                    progression: faker.random.arrayElement(Object.values(Progression)),
+                })
+                updatedAction = actionTestdata.revisedActionData(action, assignedToParticipant)
+                const notesCreator = new Participant({
+                    user: user,
+                    role: faker.random.arrayElement(assignableRoles),
+                    progression: faker.random.arrayElement(Object.values(Progression)),
```
Comment (maiaht) [inline frontend/cypress/integration/actions_spec.ts:148]: The `Finished` progression step should be filtered out for these two progressions

File: frontend/cypress/integration/actions_spec.ts [hunk 3]
```diff
@@ -124,37 +127,79 @@ describe('Actions management', () => {
...
```
Comment (maiaht) [inline frontend/cypress/integration/actions_spec.ts:191]: The `Finished` progression step should be filtered out for these two progressions

File: frontend/cypress/integration/actions_spec.ts (hunk 2 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/actions_spec.ts]: Fixed

File: frontend/cypress/integration/actions_spec.ts (hunk 3 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/actions_spec.ts]: Fixed

**Review by maiaht — APPROVED** [review]

**Pull request merged**